rotkehlchen/chain/ethereum/accounting/aggregator.py

```diff
@@ -112,6 +113,52 @@ def get_accounting_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSett
             take=1,
             method='acquisition',
         )
+        swap_key = str(HistoryEventType.TRADE) + '__' + str(HistoryEventSubType.SPEND)
+        result[swap_key] = TxEventSettings(
+            taxable=True,
+            count_entire_amount_spend=False,
+            count_cost_basis_pnl=True,
+            method='spend',
+            take=2,
+            multitake_treatment=TxMultitakeTreatment.SWAP,
+        )
+        airdrop_key = str(HistoryEventType.RECEIVE) + '__' + str(HistoryEventSubType.AIRDROP)
+        airdrops_taxable = LedgerActionType.AIRDROP in pot.settings.taxable_ledger_actions
+        result[airdrop_key] = TxEventSettings(
+            taxable=airdrops_taxable,
+            count_entire_amount_spend=False,
+            count_cost_basis_pnl=False,
+            method='acquisition',
+            take=1,
+            multitake_treatment=None,
+        )
```

Comment:
This has to respect the setting for airdrops as we do in https://github.com/rotki/rotki/blob/296f830262c3022830d84130396549a445341fd1/rotkehlchen/chain/ethereum/modules/airdrops/accountant.py#L21

Reply:
Hmmm I don't get you, what setting?

Reply:
This one that sets if the airdrops are taxable https://github.com/rotki/rotki/blob/296f830262c3022830d84130396549a445341fd1/rotkehlchen/chain/ethereum/modules/airdrops/accountant.py#L19

Reply:
Isn't this setting respected here? :thinking: 
![image](https://user-images.githubusercontent.com/63492346/210872200-810c36e5-291c-441f-9e0a-51c186dcdfa3.png)

Reply:
I would swear it was not there but yes, that one. Sorry